bedrock/firefox/templates/firefox/channel/desktop.html

```diff
@@ -96,6 +96,7 @@
           {{ ftl('firefox-channel-all-languages-and-platforms', fallback='firefox-channel-all-languages-and-builds') }}
         </a>
       </li>
+      <li><a href="https://ftp.mozilla.org/pub/firefox/nightly/">{{ ftl('firefox-channel-previous-nightly-builds') }}</a></li>
```

Comment:
This string doesn't exist in the Fluent file at `/firefox/channel.ftl`. It needs to be added there so it can be rendered on the page and localized in other languages. See https://bedrock.readthedocs.io/en/latest/l10n.html

Without a text string defined the template will display the string ID:
![image](https://user-images.githubusercontent.com/205591/89565001-b8b20580-d7d2-11ea-8882-be88ef86b350.png)

Reply:
String is added in the Fluent file at /firefox/channel.ftl